Makefile

```diff
@@ -6,4 +6,17 @@ test:
 	bats ./tests/get-test-json.bats
 	bats ./tests/get-test-full.bats
 	bats ./tests/post-test.bats
+	bats ./tests/cloud-http-get-test.bats
+	bats ./tests/cloud-http-post-test.bats
+	bats ./tests/cloud-http-get-header-test.bats
```

Comment:
I think that these tests should be ignored in PRs and only run on pushes to `staging` and maybe on a workflow triggered with intervals (cron)

Reply:
I mean, something like this to skip these tests when running from a PR:

https://github.com/infinyon/fluvio/pull/3092/commits/35bef9fe30dff63b2d68d03c50995bc4ae25f0a9

Reply:
A PR for bors support is available on: #33

Reply:
These tests shouldn't be part of integration tests because they test what is published to hub, not what is in the sources

Reply:
Good point, we should run them only on a scheduled workflow

Reply:
Already moved into its own workflow file. Will do test on push until we have green output so I can make sure its working. Then will only run it via schedule